File: ADDREV.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
	int t;
	cin>>t;
	while (t--)
	{
		string a,b;
		cin>>a>>b;  //2 numbers input as string
		reverse(a.begin(), a.end());  // reverse first number
		reverse(b.begin(), b.end());  // reverse second number
		int total = stoi(a)+stoi(b);  // stoi--string to integer
		string total_str = to_string(total);  // interger to string
		reverse(total_str.begin(), total_str.end());
		total = stoi(total_str);
		cout<<total<<"\n";
	}
	return 0;
}

File: ADDREV.py
# Problem Link : https://www.spoj.com/problems/ADDREV/

t = int(input())
while t>0:
	a,b = input().split()
	# print(a, b)
	a = int(a[::-1])  # Reverse first number
	b = int(b[::-1])  # Reverse second number
	print(int(str((a+b))[::-1]))  # Add and print reversed ans
	# print(a+b)
	t=t-1
